src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -13,179 +13,306 @@ Sending all your application errors to Sentry ensures you'll be notified in real
+### Which issues are consuming my quota? {#-what-issues-are-consuming-my-quota}
 
-To learn more and see code samples, check out:
+You can set up a query in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:
 
-- [Filtering](/platforms/javascript/configuration/filtering/)
-- [Enriching Events: Add Context](/platforms/javascript/enriching-events/context/)
+![Busiest Projects](manage-event-stream-02.png)
 
-#### GlobalHandlers
+Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest issues:
+
+![Busiest Issues](manage-event-stream-10.png)
 
-The integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default, but can be disabled through configuration. Learn more in [GlobalHandlers Integration](/platforms/javascript/configuration/integrations/default/#globalhandlers).
+Once you know where your quota is being used, you can start taking the steps below to control it.
 
-Check out additional configuration options with the [tryCatch](/platforms/javascript/configuration/integrations/default/#trycatch) and [ReportingObserver](/platforms/javascript/configuration/integrations/plugin/#reportingobserver) integrations.
+## 1. Spike Protection {#7-spike-protection}
 
-### Other SDKs
+A spike is both a **large** and **temporary** increase in error event volume. Sentry applies a dynamic rate limit to your account designed to protect you from short-term spikes.
 
-**Java** - Sentry's [Java SDK](/platforms/java/) provides integrations with common Java logging libraries. The configuration allows you to set a logging threshold determining the minimum level to capture a log message as breadcrumb. It also provides a setting to configure the minimum log level to capture an event.
+Spike protection is enabled for your organization by default, and when it's enabled, Sentry continually monitors for spikes. You can confirm that it's enabled in **Settings > Subscription**. This page can only be accessed by a Billing or Owner member of your Sentry organization.
 
-**PHP** - The `error_types` configuration option allows you to set the error types you want Sentry to monitor. Learn more in PHP [Basic Options](/platforms/php/configuration/options/#common-options).
+**We also include a one-time grace period for all paid accounts.** If an account depletes its event volume and has never previously triggered a grace period, we continue to accept events for the next three days at a maximum rate of 10,000 events per hour. If you choose not to increase your volume, Sentry will stop accepting events for the remainder of the current billing month. However, if you increase your volume, events accepted during the grace period will be counted as consumed volume.
```

Comment:
Does line 52 have anything to do with spike protection? It was with this content in the original so I haven't moved it, but it's not clear to me that it has any relation to spike protection. Can someone confirm? If not, then I'm going to move it.

Reply:
Grace period almost needs its own section. It's not related to Spike Protection however the two questions are often related for customers that experiencing large surges of events that deplete their w